src/config.js

```diff
@@ -15,9 +15,12 @@ const config = {
   ADVANCED_MODE_ENABLED: true,
   GRAPHCMS_API: 'https://api-euwest.graphcms.com/v1/ck2olkts50bo501i0h3ej7d1f/master',
   INTERCOM_APP_ID: 'a3zrft9d',
-  AUTH_CLIENT_ID: '0JKglYaNEiGxTuAAFMYbXL4m7P6Qk4eWU62LhvMv',
+//  AUTH_CLIENT_ID: '0JKglYaNEiGxTuAAFMYbXL4m7P6Qk4eWU62LhvMv',
+//  AUTH_CLIENT_SECRET:
+//  '5cNFTdJGtR0qwRYevWOMFjNIepkl6TMieZP6wdqOOIVplL0YWB8Lw5WvizU6CHiIBn21JnUd0Twc1W52MyzFRsGrV8AixFcZJpnHY3fzgiaYv9AyXS1zDbNP0xUqAxgp'
+  AUTH_CLIENT_ID: '1rftGOrlUDgZEGKFSg9yIbizOTM26N4QuOQHfBME',
   AUTH_CLIENT_SECRET:
-  '5cNFTdJGtR0qwRYevWOMFjNIepkl6TMieZP6wdqOOIVplL0YWB8Lw5WvizU6CHiIBn21JnUd0Twc1W52MyzFRsGrV8AixFcZJpnHY3fzgiaYv9AyXS1zDbNP0xUqAxgp'
+  'hpS5mRTDN8EWRgTFlDetU6mlMJw5VintTSPazuP0MUil4KyvEQnNAZHN79PObFtjTR8lL9XwTWxboXiMA19f6s4AeWNuZK7X2d1B2icmRj9q1ol5m0Qpp8RAJHVBvSji'
```

Comment:
@pragai I suppose those are dev credentials, please update back to prod so we can merge